modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFolderLocalServiceImpl.java

```diff
@@ -137,6 +139,25 @@ public ObjectFolder getObjectFolder(long companyId, String name)
 		return objectFolderPersistence.findByC_N(companyId, name);
 	}
 
+	@Override
+	public ObjectFolder getOrCreateUncategorizedObjectFolder(long companyId)
```

Comment:
As the methods to create starts with `add`, maybe we should replace with `getOrAddUncategorizedObjectFolder

Reply:
I thought about it but I don't know why I didn't change it. I think you are right. I will change it.

Reply:
Done!